python/dolma/warc/processor.py

```diff
@@ -192,12 +194,15 @@ def process_single(
                     continue
 
                 # extract text
-                doc.text = linearizer.linearize(content=decoded_content)
+                if skip_linearization:
```

Comment:
can we have a no-op linearizer instead of a boolean flag?

Reply:
Will do